deployment/helm/templates/alert.yaml

```diff
@@ -131,31 +155,26 @@ spec:
   type: ClusterIP
 ---
 
-{{- if and .Values.enablePersistentStorage (not .Values.persistentVolumeClaimName) }}
-apiVersion: v1
-kind: PersistentVolumeClaim
+{{ if and .Values.exposeui (eq .Values.exposedServiceType "Openshift" ) -}}
+apiVersion: route.openshift.io/v1
+kind: Route
 metadata:
   labels:
     app: alert
+    component: route
     name: {{ .Release.Name }}
-  name: {{ .Release.Name }}-alert-pvc
+  name: {{ .Release.Name }}-alert-exposed
   namespace: {{ .Release.Namespace }}
 spec:
-  {{ if .Values.storageClassName -}}
-  storageClassName: {{ .Values.storageClassName }}
-  {{ end -}}
-  {{ if .Values.volumeName -}}
-  volumeName: {{ .Values.volumeName }}
-  {{ end -}}
-  accessModes:
-    - ReadWriteOnce
-  resources:
-    requests:
-      storage: {{ .Values.pvcSize }}
----
-{{- end }}
-
-{{ if .Values.exposeui -}}
+  port:
+    targetPort: port-{{ .Values.alert.port }}
+  tls:
+    termination: passthrough
+  to:
+    kind: Service
+    name: {{ .Release.Name }}-alert
```

Comment:
can you please check whether an Alert service name is correct?

Reply:
It matches the name of the ClusterIP service that is always created with Alert

Reply:
```
apiVersion: v1
kind: Service
metadata:
  labels:
    app: alert
    name: {{ .Release.Name }}
  name: {{ .Release.Name }}-alert
  namespace: {{ .Release.Namespace }}
```